web/cypress/E2E_TEST_SCENARIOS.md

```
@@ -43,27 +44,27 @@ These tests verify the Monitoring Plugin functionality within the Virtualization
-| | | | |
-| `04.coo_ivt_legacy_dashboards.cy.ts` | Setting up Monitoring Plugin | 1. Setting up Monitoring Plugin | N/A | N/A |
-| `04.coo_ivt_legacy_dashboards.cy.ts` | IVT: Monitoring UIPlugin + Virtualization | 1. Virtualization perspective - Observe Menu | Virtualization | N/A |
-| `04.coo_ivt_legacy_dashboards.cy.ts` | Regression: Monitoring - Legacy Dashboards (Virtualization) | *Runs All Regression Legacy Dashboards Tests* | Virtualization | All Projects |
-| `04.coo_ivt_legacy_dashboards.cy.ts` | Regression: Monitoring - Legacy Dashboards Namespaced (Virtualization) | *Runs All Regression Legacy Dashboards Tests (Namespace)* | Virtualization | openshift-monitoring |
+| File                                 | Test Suite                                                             | Test Scenario                                             | Perspective    | Namespace Scope      |
+| ------------------------------------ | ---------------------------------------------------------------------- | --------------------------------------------------------- | -------------- | -------------------- |
+| `01.coo_ivt.cy.ts`                   | Setting up Monitoring Plugin                                           | 1. Setting up Monitoring Plugin                           | N/A            | N/A                  |
+| `01.coo_ivt.cy.ts`                   | Installation: Virtualization                                           | 1. Virtualization perspective - Observe Menu              | Virtualization | N/A                  |
+| `01.coo_ivt.cy.ts`                   | IVT: Monitoring + Virtualization                                       | _Runs BVT Monitoring Tests_                               | Virtualization | All Projects         |
+| `01.coo_ivt.cy.ts`                   | IVT: Monitoring + Virtualization - Namespaced                          | _Runs BVT Monitoring Tests (Namespace)_                   | Virtualization | openshift-monitoring |
+|                                      |                                                                        |                                                           |                |
```

markdownlint-cli2 (0.23.2)

[warning] Table column count
Expected: 5; Actual: 4; Too few cells, row will be missing data

(MD056, table-column-count)